man/journal-upload.conf.xml

```diff
@@ -60,6 +60,70 @@
         <xi:include href="version-info.xml" xpointer="v232"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>BatchMaxBytes=</varname></term>
+
+        <listitem><para>Max payload size of journal entries batch in a single HTTP request.</para>
+
+        <xi:include href="version-info.xml" xpointer="v257"/></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>BatchMaxEntries=</varname></term>
+
+        <listitem><para>Max number of journal entries in a batch in a single HTTP request.</para>
+
+        <xi:include href="version-info.xml" xpointer="v257"/></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>BatchTimeoutSec=</varname></term>
+
+        <listitem><para>Max time in seconds to wait since previous request before sending data.</para>
+
+        <xi:include href="version-info.xml" xpointer="v257"/></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>Compression=</varname></term>
+
+        <listitem><para>Compression algorithm to apply to logs data before sending.
+        Supported values are <literal>ZSTD</literal>, <literal>XZ</literal>, <literal>LZ4</literal> and <literal>NONE</literal>.
+        Default is <literal>NONE</literal></para>
```

Comment:
- Please make them lower case, when you update `compression_table`.
- Please insert a comma before `and`.
- `Default is` -> `Defaults to`
- Missing period at the and.